File: fibonacci.rb
def fibonacci(n)
  return n if n < 2
  fibonacci(n - 1) + fibonacci(n - 2)
end

start_time = Time.now

result = fibonacci(30)
puts "Result: #{result}"

end_time = Time.now
elapsed_ms = (end_time - start_time) * 1000
puts format("Time: %.3f ms", elapsed_ms)

File: fibonacci.js
function fibonacci(n) {
  if (n < 2) {
    return n;
  }
  return fibonacci(n - 1) + fibonacci(n - 2);
}

const startTime = Date.now();

const result = fibonacci(30);
console.log(`Result: ${result}`);

const endTime = Date.now();
const elapsedMs = endTime - startTime;
console.log(`Time: ${elapsedMs} ms`);
